internal/images/registry-auth.go

```diff
@@ -29,7 +29,7 @@ func (r *RegistryAuth) GetAuthFileFromSecret(ctx context.Context, namespace, nam
 	}
 	authFile, found := secret.Data[".dockerconfigjson"]
 	if !found {
-		return "", errors.New(fmt.Sprintf(".dockerconfigjson not found in %s/%s Secret", namespace, name))
+		return "", fmt.Errorf(".dockerconfigjson not found in %s/%s Secret", namespace, name)
```

Comment:
is this something that golint brought up? cause i also used 'errors.New' here and there.

Reply:
yes, Errors.New without fmt.Sprintf() is ok.

Reply:
ahhhhh it's the combination of the two that it doesn't like. ok. go figure.
Thanks again. good to know.